portfolio/src/main/webapp/index.html

```diff
@@ -59,8 +59,12 @@ <h2>Hidden Talent? </h2>
       <img src="/myImages/tournament.jpeg" alt="taekwondo" class="tournament"><br>
 
       <p>Click here to get a random quote from my favorite TV shows or fact about me:</p>
-      <button onclick="addRandomGreeting()">Click</button>
-      <div id="greeting-container"></div>
+      <button onclick="addRandomFacts()">Click</button>
+      <div id="container"></div><br>
```

Comment:
Not a great name, also name the IDs in a way that it correspond to what it is used for.
For example in this this is random-fact-container.

Reply:
Thank you for the feedback, I'll make the changes.